cypress/objects/loginPage.js

```diff
@@ -0,0 +1,39 @@
+class LoginPage {
+    visit() {
+        cy.visit('/');
+    }
+    getUserValidationIndicator() {
+        return cy.get('.Select.not-valid');
+    }
+    getSubmitButton() {
+        return cy.get('[type="submit"]');
+    }
+    getSpecificUserFromDropDown(userName) {
+        return cy.get(`[aria-label="${userName}"]`);
+    }
+    getUserFieldSelection() {
+        return cy.get('[id="loginForm.userId"]');
+    }
+    getUserRoleSelection() {
+        return cy.get('[id="loginForm.role"]');
+    }
+    getPageTitleTimesheets() {
```

Comment:
this is locator from another page (not login) so it should be in another file for timeLoggin page